project/plugins.sbt

```diff
@@ -1,10 +1,17 @@
-// Inject build variables into app code.
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
-// Code formatting.
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.2")
-// Dynamically set version based on git tags / hashes.
-addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.0.0")
-// Parallelize dependency resolution / download.
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M13-2")
-// Generate coverage reports from tests.
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
+// HACKERY: We have to hard-code the organization path in the pattern, and force-overwrite the
+// expected patterns, to handle all of the work-arounds we needed to use to get the plugins
+// publshing to Artifactory in the first place.
+val patternBase =
+  "org/broadinstitute/monster/[module](_[scalaVersion])(_[sbtVersion])/[revision]"
+
+val publishPatterns = Patterns()
+  .withIsMavenCompatible(false)
+  .withIvyPatterns(Vector(s"$patternBase/ivy-[revision].xml"))
+  .withArtifactPatterns(Vector(s"$patternBase/[module]-[revision](-[classifier]).[ext]"))
+
+resolvers += Resolver.url(
+  "Broad Artifactory",
+  new URL("https://broadinstitute.jfrog.io/broadinstitute/libs-release/")
+)(publishPatterns)
+
+addSbtPlugin("org.broadinstitute.monster" % "monster-sbt-plugins" % "0.1.0")
```

Comment:
This file will still need to be copy-pasted / included in the template repo, but at least it's mostly constant. Only the version number here will need to change if we tweak our common build parameters.